packages/sdk/src/HopBridge.ts

```diff
@@ -787,11 +788,15 @@ class HopBridge extends Base {
 
     let availableLiquidity = credit.sub(debit)
 
-    const isBondableChain = bondableChains.includes(sourceChain.slug)
-    const includePendingAmount =
-      isBondableChain && destinationChain.equals(Chain.Ethereum)
+    const unsetCommitAmount = await this.getUnsetCommitAmount(
```

Comment:
unset -> unbonded

Reply:
forgot to update sdk, thanks for catching that